sql/changes/mc/acc-trans-migration.sql

```diff
@@ -132,6 +132,16 @@ BEGIN
          AND NOT EXISTS (select 1 from cr_report_line cl
                            join lines_to_delete ltd on cl.ledger_id = ltd.assoc
                           where ltd.id = crl.ledger_id);
+  UPDATE cr_report_line crl
+     SET our_balance = our_balance
+                     + (select our_balance from cr_report_line cl
+                           join lines_to_delete ltd on cl.ledger_id = ltd.id
+                          where ltd.id = crl.ledger_id or ltd.assoc = crl.ledger_id),
```

Comment:
Are you sure the `assoc` is correct here? It's not the line being deleted, so I would expect either the sum to be incorrect (sometimes) or multiple lines to be returned, which isn't allowed with this nested select.